server/controllers/expenses.js

```diff
@@ -35,7 +36,17 @@ export const create = (req, res, next) => {
 /**
  * Get an expense.
  */
-export const getOne = (req, res) => res.json(req.expense.info);
+export const getOne = (req, res) => {
+  auth.canEditExpense(req, res, (e, canEditExpense) => {
```

Comment:
Seems weird to call auth from controller. We probably want to decorate req.remoteUser with this data?